### ISSUE TITLE
Move MIX2 to FPWD

### ISSUE THREAD

**Issue by carlosjoan91** [w3c/webappsec-mixed-content#25]
We discussed [Mixed Content Level 2](https://w3c.github.io/webappsec-mixed-content/level2.html) during [TPAC](https://github.com/w3c/webappsec/blob/master/meetings/2019/2019-09-TPAC-minutes.md#mix2), and there was some amount of consensus (and no objections) in moving mixed content in that direction. Chrome is currently planning to start autoupgrades for audio and video mixed content starting in version 80 (to be released in February 2020).

This is a CfC for moving the existing ED to FPWD. Please add any thoughts (preferably here in the issue).

**Comment by carlosjoan91** [comment]
I created an FPWD version of the ED document in https://github.com/w3c/webappsec-mixed-content/pull/24

**Comment by annevk** [comment]
I find it very hard to review as it's not clear compared to Level 1 what is removed, replaced, or added.

Also, I'd expect a Level 2 document to be self-contained and not written as a set of updates to a prior document.

**Comment by carlosjoan91** [comment]
Do you think copying the non-changed stuff from Level 1 would be better? That way the doc would be self contained (and can completely replace level 1 once published).

**Comment by annevk** [comment]
Yeah, that's definitely where we want to end up I think.

**Comment by carlosjoan91** [comment]
I sent a PR (https://github.com/w3c/webappsec-mixed-content/pull/30) that merges the still relevant concepts from MIX onto MIX2, could you take a look and see if addresses your concerns @annevk ?

**Comment by annevk** [comment]
Shall we move forward with this? Or at least make the editor's draft of Mixed Content be the Level 2 version.

**Comment by carlosjoan91** [comment]
Happy to move forward if there are no objections, I'll start by making MIX2 be the editor's draft like you suggest (to give folks a chance for more comments since this was dormant for a while) then push it to FPWD if nothing comes in.

**Comment by wseltzer** [comment]
It would be nice to have the "Latest published version:" link go someplace. Currently, there's nothing at  https://www.w3.org/TR/mixed-content-2/

**Comment by carlosjoan91** [comment]
Since MIX2 has been fairly stable and no comments have come in, my guess is we can go ahead and start the Transition Request process to FPWD. I'll add this to the next meeting agenda to do one last check before starting it.

In the meantime, I'm trying to figure out how to get bikeshed to build without a "Latest published version" link, since there is none other than the editor's draft.

**Comment by sideshowbarker** [comment]
Closing this as I believe that with the move of the Mixed Content spec to permanent CRD status — and when the changes in #57 are merged to help clarify that the spec is now being continuously maintained (rather than replaced by a new version) — it’s no longer necessary or useful to do a FPWD for a new rev of the spec (when the changes in #57 are merged there will in fact no longer be any mention at all of the current spec being Level 2; instead it’s unversioned).
